cni-plugin/pkg/dataplane/windows/dataplane_windows.go

```diff
@@ -1024,7 +1024,7 @@ func (d *windowsDataplane) CleanUpNamespace(args *skel.CmdArgs) error {
 	d.logger.Infof("Attempting to delete HNS endpoint name : %s for container", epName)
 
 	if cri.IsDockershimV1(args.Netns) {
-		err = hns.DeprovisionEndpoint(epName, args.Netns, args.ContainerID)
+		err = hns.RemoveHnsEndpoint(epName, args.Netns, args.ContainerID)
```

Comment:
same for this - did the name need to change?

At a minimum, HNS is an acronym so we should probably name this `RemoveHNSEndpoint` to be consistend with idiomatic golang naming.

Reply:
That's from upstream, they change the name of the methods. 
https://github.com/containernetworking/plugins/commit/ec75bb8587443894405d0c0d8ee328b8dbaba761#diff-5785f41c956eab73b527ca6708c2af5457ebda462fd75821c6b5e1b47d6f4062

Reply:
oh right I see, yes ok this is fine then